blobs.go

```diff
@@ -61,6 +61,12 @@ type Descriptor struct {
 	// depend on the simplicity of this type.
 }
 
+// Descriptor returns the descriptor, to make it satisfy the Describable
```

Comment:
> This does not satisfy the Describable interface. Describable must describe itself (in this case, the descriptor), not the thing it is describing.

There's a common use case where some code has a slice of descriptors and wants to add those descriptors to a manifest through a `ManifestBuilder`. Since `ManifestBuilder.AppendReference` takes a `Describable`, not having this method defined would force the creation of a wrapper type just to return a descriptor.

Instead of having `Descriptor` satisfy `Describable`, would you be okay with changing `AppendReference` to take a `Descriptor`?